packages/integration/src/bitbucketServer/config.ts

```diff
@@ -73,6 +85,19 @@ export function readBitbucketServerIntegrationConfig(
     apiBaseUrl = `https://${host}/rest/api/1.0`;
   }
 
+  if (config.getOptionalConfig('headers')) {
+    const headers: HeadersKV = {};
+    for (const itemKey of config.getConfig('headers').keys()) {
+      headers[`${itemKey}`] = config.getString(`headers.${itemKey}`);
+    }
```

Comment:
```suggestion
    for (const [key, value] of config.getConfig('headers').entries()) {
      headers[key] = value;
    }
```

This should work, right?

Reply:
this looks much better, thanks for the suggestion

Reply:
actually the config object does not have that method, I only see methods that allow me to query one KV at a time or `.keys()` that lets me iterate over all the keys to copy over the KVs to the request headers object.